src/interfaces/forms.d.ts

```diff
@@ -1,5 +1,20 @@
 // <reference types="react" />
 
+export interface IFormProps extends React.InputHTMLAttributes<HTMLFormElement> {
```

Comment:
`InputHTMLAttributes` ?

Reply:
changed to `React.HTMLAttributes<HTMLFormElement>`